engine/crates/federated-graph/src/federated_graph/v1.rs

```diff
@@ -230,6 +230,41 @@ pub struct Override {
     /// Points to a subgraph referenced by name, but this is _not_ validated to allow easier field
     /// migrations between subgraphs.
     pub from: OverrideSource,
+    #[serde(default)]
+    pub label: OverrideLabel,
```

Comment:
would use an enum here, hard to know what will be in this `label` in the future.

Reply:
Yes this struct felt a bit awkward in different places, let's do that.

Reply:
I mean transform `OverrideLabel` into an enum, maybe doesn't matter

Reply:
yep that's how I understood it. I'm wondering if we should already have a fallback variant for forward compatibility.

Reply:
Up to you, won't do anything with it engine-side 🤷 
would avoid from_sdl -> to_sdl to change anything though